staging/src/k8s.io/apiserver/pkg/endpoints/handlers/create.go

```diff
@@ -141,7 +141,8 @@ func createHandler(r rest.NamedCreater, scope *RequestScope, admit admission.Int
 
 		ae := request.AuditEventFrom(ctx)
 		admit = admission.WithAudit(admit, ae)
-		audit.LogRequestObject(ae, obj, scope.Resource, scope.Subresource, scope.Serializer)
+		objGV := schema.GroupVersion{Group: gvk.Group, Version: gvk.Version}
```

Comment:
construct like this, and move above the `if !scope.AcceptsGroupVersion(gvk.GroupVersion())` line and use objGV as the param to that function as well

```suggestion
		objGV := gvk.GroupVersion()
```

Reply:
You are very careful.
respect!